frontend/src/scenes/insights/LineGraph/LineGraph.tsx

```diff
@@ -355,7 +357,9 @@ export function LineGraph(props: LineGraphProps): JSX.Element {
                                         forceEntitiesAsColumns={isHorizontal}
                                         hideInspectActorsSection={!(onClick && showPersonsModal)}
                                         groupTypeLabel={
-                                            aggregationLabel(seriesData?.[0]?.action?.math_group_type_index).plural
+                                            aggregationGroupTypeIndex === -1
```

Comment:
We don't use -1 anywhere else. `number | null` would be a better type, along with checking it as `== undefined` (see frontend/src/scenes/trends/trendsLogic.ts)